app/js/lib/form_validation.js

```diff
@@ -40,37 +40,74 @@ var jQuery = require( 'jquery' ),
 
 	AmountValidator = {
 		validationUrl: '',
-		postFunction: null,
+		sendFunction: null,
 		validate: function ( formValues ) {
 			var postData = {
 				amount: formValues.amount,
 				paymentType: formValues.paymentType
 			};
-			return this.postFunction( this.validationUrl, postData, null, 'json' );
+			return this.sendFunction( this.validationUrl, postData, null, 'json' );
 		}
 	},
 
-	createAddressValidator = function ( validationUrl, postFunction ) {
+	BankDataValidator = {
+		validationUrlForSepa: '',
+		validationUrlForNonSepa: '',
+		sendFunction: null,
+		validate: function ( formValues ) {
+			var data, validationUrl;
+			if ( formValues.paymentType !== 'BEZ' ) {
+				return {
+					status: 'OK'
+				};
+			}
+			if ( formValues.debitType === 'sepa' ) {
+				data = {
+					iban: formValues.iban
+				};
+				validationUrl = this.validationUrlForSepa;
+			} else {
+				data = {
+					accountNumber: formValues.accountNumber,
+					bankCode: formValues.bankCode
+				};
+				validationUrl = this.validationUrlForNonSepa;
+			}
+			return this.sendFunction( validationUrl, data, null, 'json' );
+		}
+	},
+
+	createAddressValidator = function ( validationUrl, sendFunction ) {
 		return objectAssign( Object.create( AddressValidator ), {
 			validationUrl: validationUrl,
-			postFunction: postFunction || jQuery.post
+			sendFunction: sendFunction || jQuery.post
 		} );
 	},
 
 	/**
 	 *
 	 * @param {string} validationUrl
-	 * @param {Function} postFunction jQuery.post function or equivalent
+	 * @param {Function} sendFunction jQuery.post function or equivalent
 	 * @return {AmountValidator}
 	 */
-	createAmountValidator = function ( validationUrl, postFunction ) {
+	createAmountValidator = function ( validationUrl, sendFunction ) {
 		return objectAssign( Object.create( AmountValidator ), {
 			validationUrl: validationUrl,
-			postFunction: postFunction || jQuery.post
+			sendFunction: sendFunction || jQuery.post
+		} );
+	},
+
+	createBankDataValidator = function ( validationUrlForSepa, validationUrlForNonSepa, sendFunction ) {
+		return objectAssign( Object.create( BankDataValidator ), {
+			validationUrlForSepa: validationUrlForSepa,
+			validationUrlForNonSepa: validationUrlForNonSepa,
+			sendFunction: sendFunction || jQuery.get
```

Comment:
Don't know. The route just accepts GET, but I could also change the route instead of jQuery method.